master/docs/manual/installation/worker.rst

```diff
@@ -183,6 +183,28 @@ Other Worker Configuration
                keepalive, usepty, umask=umask, maxdelay=maxdelay,
                unicode_encoding='utf-8', allow_shutdown='signal')
 
+.. _Worker-TLS-Config:
+
+Worker TLS Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+``connection_string``
+    For TLS connections to the master the ``connection_string``-argument must be used to the Worker-initializer. ``buildmaster_host`` and ``port`` must then be ``None``.
+
+    ``connection_string`` will be used to create a client endpoint with clientFromString_. An example of ``connection_string`` is ``"TLS:buildbot-master.com:9989"``.
+
+    See more about how to formulate the connection string in ConnectionStrings_.
+
+    Make sure the worker trusts the masters certificate. If you have an self-signed certificate the trustRoot parameter can be used: ``'TLS:buildbot-master.com:9989:trustRoots=/dir-with-ca-certs'``. It must point to a directory with PEM-encoded certificates in files with file ending .pem.
+
+.. code-block:: python
+
+    s = Worker(None, None, workername, passwd, basedir,
+               keepalive, connection_string='TLS:buildbot-master.com:9989')
+
```

Comment:
We should probably  explain that :

```
 s = Worker('master', 9989, workername, passwd, basedir,
                      keepalive)
```

 is equivalent to

``` 
    s = Worker(None, None, workername, passwd, basedir,
                      keepalive, connection_string="tcp:master:9989")
```

Reply:
I've added that too. And also expanded about the trustRoots.